src/kubernetes.cc

```diff
@@ -757,13 +761,23 @@ struct Watcher : public std::thread {
         LOG(ERROR) << name_ << " => Unlocking completion mutex";
       }
       service_->stop();
+    } else {
+      silence_timer_->RunAsyncAfter(std::chrono::seconds(
```

Comment:
`silence_timer_` doesn't always exists - need to check whether it is `nullptr`

Reply:
Done.